Makefile

```diff
@@ -90,6 +93,10 @@ ifneq ($(wildcard /.dockerenv), )
 	go test -check.v -check.timeout=$(TEST_TIMEOUT) timeout=$(TEST_SUITE_TIMEOUT) $(TESTFLAGS) github.com/docker/containerd/integration-test
 endif
 
+bench: all validate
```

Comment:
Why not? building shim (part of all) is necessary for bench and code should be formatted.